torch/fx/experimental/sym_node.py

```diff
@@ -1097,7 +1096,7 @@ def sizes_strides_impl(self, sizes, strides):
         try:
             out = func(size_exprs, stride_exprs)
         except Exception:
-            log.warning("failed to eval %s(%s, %s)", method, size_exprs, stride_exprs)
+            sym_node_log.warning("failed to eval %s(%s, %s)", method, size_exprs, stride_exprs)
```

Comment:
Would you kindly revert the changes for the warnings? The warnings should be emitted by default with or without the sym_node artifact being enabled.